CMakeLists.txt

```diff
@@ -40,18 +40,24 @@ option(SWIFT_BUILD_TOOLS
     "Build the Swift compiler and other tools"
     TRUE)
 
-option(SWIFT_BUILD_STDLIB
-    "Build the Swift standard library (independent of the SDK headers)"
-    TRUE)
-
```

Comment:
Sorry, the names are confusing here.  This option controls the build of the core standard library (`stdlib/public/CMakeLists.txt`):

```
if(SWIFT_BUILD_STDLIB)
  # These must be kept in dependency order so that any referenced targets
  # exist at the time we look for them in add_swift_*.
  add_subdirectory(SwiftShims)
  add_subdirectory(runtime)
  add_subdirectory(stubs)
  add_subdirectory(core)
  add_subdirectory(SwiftOnoneSupport)
  add_subdirectory(Reflection)
  add_subdirectory(SwiftRemoteMirror)
endif()
```

The variables allow building just the compiler, or the compiler + core library, or just the SDK overlay using an existing compiler.